pkg/piecestore/psserver/server_test.go

```diff
@@ -198,17 +198,17 @@ func TestRetrieve(t *testing.T) {
 			respSize:  5,
 			allocSize: 5,
 			offset:    0,
-			content:   []byte("butts"),
-			err:       "rpc error: code = Unknown desc = argError: invalid id length",
+			content:   []byte("xyzwq"),
+			err:       "rpc error: code = Unknown desc = piecestore error: invalid id length",
 		},
 		{ // server should err with nonexistent file
 			id:        "22222222222222222222",
 			reqSize:   5,
 			respSize:  5,
 			allocSize: 5,
 			offset:    0,
-			content:   []byte("butts"),
-			err:       fmt.Sprintf("rpc error: code = Unknown desc = retrieve error: stat %s: no such file or directory", path.Join(TS.s.DataDir, "/22/22/2222222222222222")),
+			content:   []byte("xyzwq"),
+			// err:       fmt.Sprintf("rpc error: code = Unknown desc = retrieve error: stat %s: no such file or directory", path.Join(TS.s.DataDir, "/22/22/2222222222222222")),
```

Comment:
can be removed?

Reply:
I wasn't sure whether it's worth fixing. All the tests here need to be rewritten anyways.